packages/xo-server/src/api/host.js

```diff
@@ -2,6 +2,22 @@ import { format } from 'json-rpc-peer'
 
 // ===================================================================
 
+export function disableCompletely({ host }) {
```

Comment:
`clearHost`?

Reply:
Or even `maintenanceMode` (with a boolean to turn it ON and OFF) so it's easier to understand, especially for people coming from XenCenter/XCP-ng Center who are used to that feature.